.github/workflows/update-policy.yml

```diff
@@ -16,7 +16,7 @@ env:
 jobs:
   update-templates:
     name: Update Policy Library
-    if: github.repository == 'Azure/ALZ-Bicep'
+    #if: github.repository == 'Azure/ALZ-Bicep'
```

Comment:
Pop this back 😁

Reply:
Still doing some work :) Roger wilco

Reply:
Reverted back to `main`. Ready for review